modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -865,8 +865,12 @@ private TableColumn<String, String> initSortTestStructure() {
         assertEquals(-1, tv.getSelectionModel().getSelectedIndex());
     }
 
-    @Ignore("Started failing recently, but manual tests seem to indicate the functionality is intact")
-    @Test public void test_rt17522_focusShouldMoveWhenItemAddedAtFocusIndex() {
+    /**
```

Comment:
We can remove this comment. If needed, we can get this information from looking at file revision history.

Reply:
I thought a bit of history won't hurt in case this test starts failing on some platform.  Will avoid adding similar comments in the future.   Thanks!